vercel.json

```diff
@@ -1,5 +1,10 @@
 {
 	"github": {
 		"silent": false
+	},
+	"functions": {
+		"src/pages/terraform/cdktf/**": {
+			"memory": 2048
+		}
```

Comment:
Docs for posterity
https://vercel.com/docs/project-configuration#project-configuration/functions/value-definition